src/components/Restoration/ApprovalType.vue

```diff
@@ -3,6 +3,7 @@
     <ApprovalTypeShared
       class="pa-8"
       :courtOrderNumber="getRestoration.courtOrder.fileNumber"
+      :approvedByCourtOrder="approvedByCourtOrder"
```

Comment:
This is needed to set the initial state of the (shared) ApprovalType component. Without this, the court order option (and its value) were not being restored/displayed.